### ISSUE TITLE
uploading newly compressed images

### ISSUE THREAD

**Issue by a-murry** [uofo/reverse-archaeology#65]
@ebrelsford is there a quick way to plug back in the images Ron compressed for bios, artifact sheets, slideshows, etc)? Or will I need to bring them back into github with different names, and re-assign the image links in each of the respective prose sheets?

**Comment by a-murry** [comment]
compressed images are here:

[](https://www.dropbox.com/sh/qz84a2eaal2sapx/AAC7-Aqg4L9gjjGNHwVIF_UIa?dl=0)

**Comment by a-murry** [comment]
[https://www.dropbox.com/sh/qz84a2eaal2sapx/AAC7-Aqg4L9gjjGNHwVIF_UIa?dl=0](url)

**Comment by ebrelsford** [comment]
@a-murry @rmorrison28 yes I will be able to easily upload the whole batch. Two things:

 1. It seems that the images have been compressed so far down that they're blank?! Am I missing something?
 2. These are all images from [this folder](https://github.com/uofo/reverse-archaeology-content/tree/gh-pages/images), not a subfolder of that, right? It seems that's the case but I'm just double-checking.

**Comment by a-murry** [comment]
@rmorrison28 I am also seeing the blank images.
